Formula/picat.rb

```diff
@@ -18,7 +19,12 @@ class Picat < Formula
   end
 
   def install
-    system "make", "-C", "emu", "-f", "Makefile.mac64"
+    makefile = "Makefile.mac64"
```

Comment:
Does this work?
```suggestion
    makefile = "Makefile.#{os}64"
```

Reply:
I'm afraid not:
```
Error: An exception occurred within a child process:
  NameError: undefined local variable or method `os' for #<Formulary::FormulaNamespacee2f92215c94f9fdd0f53cc75e8b9e82e::Picat:0x00007fb0423cfb50>
```
Not seeing `os` or anything along those lines in https://rubydoc.brew.sh/ either, though I can't be sure I didn't miss something...

Reply:
Maybe it's `OS`? Sorry, I'm doing this from memory on my phone.

Reply:
Sorry, that doesn't work either. I went through all the core formulae that use either `#{os}` or `#{OS}` in some form, and I haven't found an exception to the following:
* `#{os}` -- local variable along the lines of:
```
os = "darwin"
on_linux { os = "linux" }
```
* `#{OS}` -- only ever deferenced as `#{OS.kernel_version}`

Given all that, I'm inclined to keep things as they are.

Reply:
Yeah, sounds fine. I thought there was a method.